.vscode/settings.json

```
@@ -24,5 +24,9 @@
   "eslint.workingDirectories": [
     "./apps/db-cli",
     "."
-  ]
+  ],
+  "jest.autoRun": {
+    "watch": true,
+    "onStartup": ["all-tests"]
+  }
```

```suggestion
  "jest.autoRun": {
    "watch": false,
    "onSave": "test-file",
    "onStartup": []
  }
```

Motivations:
- Do not run all the tests at startup because this may take too much time and not be required by the developer.
-`watch: true` will runs the tests each time a component or test file is saved. This is not efficient from a performance stand point. Instead, we should use `watch: false` and `onsave: "test-file"` to trigger tests only when saving test files.